Tests/SentryTests/Integrations/SentryPerformanceTrackerTests.swift

```diff
@@ -0,0 +1,219 @@
+import XCTest
+
+class SentryPerformanceTrackerTests: XCTestCase {
+    
+    private static let dsnAsString = TestConstants.dsnAsString(username: "SentryPerformanceTrackerTests")
+    
+    private class Fixture {
+
+        let someTransaction = "Some Transaction"
+        let someOperation = "Some Operation"
+        
+        func getSut(initSDK: Bool = false) -> SentryPerformanceTracker {
+            if initSDK {
+                SentrySDK.start { options in
+                    options.dsn = SentryPerformanceTrackerTests.dsnAsString
+                    options.debug = true
+                    options.diagnosticLevel = SentryLevel.debug
+                }
+            }
+            return  SentryPerformanceTracker()
+        }
+
+        func getSpans(tracker: SentryPerformanceTracker) -> [SpanId: Span] {
+            let result = Dynamic(tracker).spans as [SpanId: Span]?
+            return result!
+        }
+        
+        func getStack(tracker: SentryPerformanceTracker) -> [Span] {
+            let result = Dynamic(tracker).activeStack as [Span]?
+            return result!
+        }
```

Comment:
`l`: The purpose of the fixture is only to generate test objects, but it is not a place to put helper functions. You can just move these functions to `SentryPerformanceTrackerTests` and make them private there.